tensorflow/lite/tools/evaluation/tasks/coco_object_detection/BUILD

```diff
@@ -13,7 +13,10 @@ py_binary(
     python_version = "PY3",
```

Comment:
@freedomtan here is the internal error , can you please check 

ERROR: lite/tools/evaluation/tasks/coco_object_detection/BUILD:11:10: no such target '/lite/tools/evaluation/proto:preprocessing_steps_py': target 'preprocessing_steps_py' not declared in package 'lite/tools/evaluation/proto' defined by lite/tools/evaluation/proto/BUILD and referenced by 'lite/tools/evaluation/tasks/coco_object_detection:preprocess_coco_minival'

Reply:
@rthadur That is a known issue. I sent a PR (#38833) for that. Since @multiverse-tf has a better fix, I closed my PR. Hopefully, that will be fixed soon.

Reply:
rebased to master and removed the `proto:preprocessing_steps` I added because it was added by other commit already

Reply:
@freedomtan still failing same error , can you please check

Reply:
close this, d76bbee1341 already fixed the problem :-)